index.html

```diff
@@ -39,94 +43,111 @@ <h1 class="greet">Hey There. <br> I’m Alex</h1>
-      <p class="multi-paragraph-card">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-      <ul class="professional-list">
-        <li>html</li>
-        <li>bootstrap</li>
-        <li>Ruby</li>
-      </ul>
-      <button class="prof-btn">See Project</button>
-    </div>
-
-    <div class="professional-art art3">
-      <h2 class="prof-header">Profesional Art <br> Printing Data</h2>
-      <p class="multi-paragraph-card">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-      <ul class="professional-list">
-        <li>html</li>
-        <li>bootstrap</li>
-        <li>Ruby</li>
-      </ul>
-      <button class="prof-btn">See Project</button>
-    </div>
-
-    <div class="professional-art art4">
-      <h2 class="prof-header">Profesional Art <br> Printing Data</h2>
-      <p class="multi-paragraph-card">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-      <ul class="professional-list">
-        <li>html</li>
-        <li>bootstrap</li>
-        <li>Ruby</li>
-      </ul>
-      <button class="prof-btn">See Project</button>
-    </div>
-
-    <div class="professional-art art5">
-      <h2 class="prof-header">Profesional Art <br> Printing Data</h2>
-      <p class="multi-paragraph-card">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-      <ul class="professional-list">
-        <li>html</li>
-        <li>bootstrap</li>
-        <li>Ruby</li>
-      </ul>
-      <button class="prof-btn">See Project</button>
+      <div class="line"></div>
     </div>
+    <div class="work-container">
+      <div class="multi-post">
+        <img src="icons/images/Img Placeholder1.png" alt="illustration image">
+        <div>
+          <h2 id="multi-head-two">Multi-Post Stories</h2>
+          <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required. 
+            has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+          <ul>
+            <li>css</li>
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button class="multi-btn">See Project</button>
+        </div>
+      </div>
+  
+      <div class="professional-art art1 prt">
+        <h2 class="prof-header">Profesional Art <br> Printing Data</h2>
+        <p class="multi-paragraph-card">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul class="professional-list">
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button class="prof-btn">See Project</button>
+  
+      </div>
```

Comment:
- [ ] For the project cards, the Figma template shows us 2 states for each card, the default state and the hover state. 

## Standard (default) state:
![image](https://user-images.githubusercontent.com/50721479/213470374-0e9c41f3-cb22-44fe-86bb-1510de71cf1d.png)

## Hover state: (you can see the cursor on it)
![image](https://user-images.githubusercontent.com/50721479/213470446-f49aee56-127a-4c0d-a304-25575fb075c2.png)

Kindly update your cards to look like the first picture in normal state and when hovered they should look like the 2nd picture (the button shows and the image doesn't have a dark overlay on top of it) 💯 This is important because you need to show the button and the project information on all the cards for a later project 👍